### PR TITLE
Clarify scope of parameters

### PR DESCRIPTION
**Pull request by rhummelmose**
**What type of PR is this?**
/kind documentation
**What this PR does / why we need it**:
Clarify scope of parameters to avoid users passing resource ids instead of names

### CODE REVIEW

File: deploy/example/e2e_usage.md [hunk 1]
```diff
@@ -22,12 +22,13 @@ metadata:
   name: blob
 provisioner: blob.csi.azure.com
 parameters:
-  resourceGroup: EXISTING_RESOURCE_GROUP
-  storageAccount: EXISTING_STORAGE_ACCOUNT
+  resourceGroup: EXISTING_RESOURCE_GROUP # Resource group name, not resource id
```
Comment (andyzhangx) [inline deploy/example/e2e_usage.md:25]: `EXISTING_RESOURCE_GROUP_NAME`

File: deploy/example/e2e_usage.md [hunk 2]
```diff
@@ -22,12 +22,13 @@ metadata:
...
+  storageAccount: EXISTING_STORAGE_ACCOUNT # Storage account name, not resource id
```
Comment (andyzhangx) [inline deploy/example/e2e_usage.md:26]: `EXISTING_STORAGE_ACCOUNT_NAME`

File: deploy/example/e2e_usage.md (hunk 1 above)
Comment (andyzhangx) [reply deploy/example/e2e_usage.md:25]: and remove comments

**Review by andyzhangx — COMMENTED** [review]
/ok-to-test

File: deploy/example/e2e_usage.md (hunk 2 above)
Comment (andyzhangx) [reply deploy/example/e2e_usage.md:26]: and remove comments

Comment (rhummelmose) [reply deploy/example/e2e_usage.md]: Done

File: deploy/example/e2e_usage.md (hunk 1 above)
Comment (rhummelmose) [reply deploy/example/e2e_usage.md]: Done

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Pull request merged**